conn.go

```diff
@@ -121,6 +121,13 @@ type ReadBatchConfig struct {
 	// ReadUncommitted makes all records visible. With ReadCommitted only
 	// non-transactional and committed records are visible.
 	IsolationLevel IsolationLevel
+
+	// The amount of time for the broker while waiting to hit the min/max byte
+	// targets.  This setting is independent of any network-level timeouts or
+	// deadlines.
+	//
+	// For backward compatibility, the default value here will todo
+	MaxWaitTime time.Duration
```

Comment:
Since we have this field named `MaxWait` in other places, how about sticking to that name to keep the APIs consistent?

Reply:
SGTM.  Also looks like I didn't finish the docs. 🤦